src/wallet/service_test.go

```diff
@@ -292,9 +344,27 @@ func TestServiceCreateAndScanWallet(t *testing.T) {
 	}
 }
 
+func TestServiceNewAddressDisabledWalletAPI(t *testing.T) {
+	dir := prepareWltDir()
+	s, err := NewService(dir, true)
+	require.NoError(t, err)
+	dirIsEmpty(t, dir)
+
+	// get the default wallet id
+	var id string
```

Comment:
remove this loop, does nothing, just check s.wallets is empty

Reply:
Fixed.